sycl/unittests/pi/cuda/test_commands.cpp

```diff
@@ -30,7 +30,7 @@ struct CudaCommandsTest : public ::testing::Test {
 
   void SetUp() override {
     // skip the tests if the CUDA backend is not available
-    if (!plugin) {
+    if (!plugin.has_value()) {
```

Comment:
why do you need .has_value()? there is a conversion to bool defined: https://en.cppreference.com/w/cpp/utility/optional/operator_bool

Reply:
Frankly it was just a personal preference when using `std::optional`. @AidanBeltonS Feel free to scrap the `.has_value()`.

Reply:
This is a NIT, feel free to leave it as is.